mkdocs.yml

```diff
@@ -223,6 +223,7 @@ nav:
                   - Resources Mirroring with CLI: ./bnb-greenfield/for-developers/tutorials/cross-chain-integration/mirror.md
                   - Integrating BSC Smart Contracts with Greenfield Projects: ./bnb-greenfield/for-developers/tutorials/cross-chain-integration/dapp-integration.md
                   - Mirroring FAQ: ./bnb-greenfield/for-developers/tutorials/cross-chain-integration/mirroring-faqs.md
+                  - Contract as Bucket Owner: ./bnb-greenfield/for-developers/tutorials/cross-chain-integration/demo-contract-as-bucket-owner.md
```

Comment:
I think the order is also important, `Mirroring FAQ` should always be the last one.

Reply:
fixed